src/utils/tassign.ts

```diff
@@ -0,0 +1,3 @@
+export function tassign<T extends U, U>(target: T, ...source: U[]): T {
```

Comment:
Thoughts on making `tassign` it's own NPM package? can see this being useful outside of `ng2-redux`.